packages/web-components/src/components/cbp-toast/cbp-toast.scss

```diff
@@ -0,0 +1,149 @@
+/**
+ * @Prop   --cbp-toast-background: var(--cbp-color-info-dark);
+ * @Prop   --cbp-toast-background-dark: var(--cbp-color-info-lighter);
+ * @Prop   --cbp-toast-color: var(--cbp-color-text-lighter);
+ * @Prop   --cbp-toast-color-dark: var(--cbp-color-text-darker);
+ * @Prop   --cbp-toast-color-content: var(--cbp-color-text-lighter);
+ * @Prop   --cbp-toast-color-content-dark: var(--cbp-color-text-darkest);
+ * @Prop   --cbp-toast-sidebar: var(--cbp-color-info-darker);
+ * @Prop   --cbp-toast-sidebardark: var(--cbp-color-info-light);
+ * @Prop   --cbp-toast-sidebar-icon: var(--cbp-color-text-light);
+ * @Prop   --cbp-toast-sidebar-icon-dark: var(--cbp-color-text-darker);
+ */
+ :root{
+ 
+  --cbp-toast-background: var(--cbp-color-info-dark);
+  --cbp-toast-background-dark: var(--cbp-color-info-lighter);
+  --cbp-toast-color: var(--cbp-color-text-lighter);
+  --cbp-toast-color-dark: var(--cbp-color-text-darker);
+  --cbp-toast-color-content: var(--cbp-color-text-lighter);
+  --cbp-toast-color-content-dark: var(--cbp-color-text-darkest);
+  --cbp-toast-sidebar: var(--cbp-color-info-darker);
```

Comment:
all of these are colors and need color in the name.

Reply:
updated to --cbp-toast-color-bg-sidebar & --cbp-toast-color-icon-sidebar